contrib/examples/actions/mistral-basic-rename.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Unrelated to this change but a thought - 

could we infer the action object from the mistral workflow?